src/components/Radio.tsx

```diff
@@ -2,36 +2,48 @@ import React, { useState } from 'react';
 import { View, Text, StyleSheet } from 'react-native';
 import { RadioButton } from 'react-native-paper';
 
-const Radio = ({ title, check }: any) => {
-  const [checked, setChecked] = useState(check);
+const Radio = ({ title, setValue,disabled }: any) => {
+  const [checked, setChecked] = useState(false);
```

Comment:
Remove this state, pass `checked` value through the props, and use it as a value on the RadioButton.